app/latigo/scheduler/__init__.py

```diff
@@ -226,15 +198,8 @@ def run(self):
 
         while not done:
             logger.info(
-                f"Next prediction will occur at {self.continuous_prediction_timer.closest_start_time()} (in {human_delta(self.continuous_prediction_timer.time_left())})"
+                f"Next prediction will occur at {self.continuous_prediction_timer.closest_start_time()} "
+                f"(in {human_delta(self.continuous_prediction_timer.time_left())})"
```

Comment:
```suggestion
                "Next prediction will occur at %s (in %s)", 
                self.continuous_prediction_timer.closest_start_time(),
                human_delta(self.continuous_prediction_timer.time_left())
```

Strings substitution with `%` is a recommended way for logging.

Reply:
done